### ISSUE TITLE
I want to use filters in category page So I can limit the products visible in the category page

### ISSUE THREAD

**Issue by rmakara** [DivanteLtd/shopware-pwa#5]
**Context**
After opening specific category I want to use filters for narrowing the list of the products to make it easier to find the ones, that I am interested in.

**Dependencies**
* Blocked by #4 
* Blocked by #7 

**Acceptance criteria**
* I can see a list of available filters for the current category.
* I can filter by [...] (to split by each filters to another stories?).
* When I use a filter then products list gets updated (to extract to another story?).
* I can use a few filters at once (to extract to another story?).
* I can filter by nested field (not a field of top level object).

**Comment by rmakara** [comment]
**Done:**
* Getting filters from Shopware.
* Displaying available filters.

**To do:**
* Applying filters to URL and getting filtered data from Shopware.
